runs/2017/pyladies-praha-jaro/info.yml

```diff
@@ -62,6 +62,8 @@ plan:
   slug: vcs
   date: 2017-04-24
   materials:
+  - lesson: git/git-collaboration-2in1
+  - lesson: beginners/files
```

Comment:
Pozor na to, že lekce o souborech navazuje na výjimky – `with` je vysvětlený přes `try/finally`